wildfly-getting-started-archetype/src/main/resources/archetype-resources/src/test/java/org/wildfly/security/examples/test/SampleIT.java

```diff
@@ -3,14 +3,15 @@
 #set( $symbol_escape = '\' )
 package org.wildfly.security.examples.test;
```

Comment:
@darranl That's not related to the intent of this PR but the generated example should also use the package instead of this hard-coded name